### DISCUSSION TITLE
Typo in CHANGELOG.md "Windows 11 or Windows 11 style"

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#560] [General]
I really enjoy following the coding on this great project, spotted a small typo.
From commit[ Ability to choose Snap Assist style](https://github.com/valinet/ExplorerPatcher/commit/94db02bde04b676d897736773d262d5468a0df39)

* Ability to choose look of Snap Assist (window list when snapping a window): Windows 11 or Windows 11 style (.11)

**Comment by valinet** [comment]
Yeah, Windows 10 or 11 indeed. I will correct that at some point, thanks for letting me know.
